_includes/footer/footer.njk

```diff
@@ -1,8 +1,25 @@
-<footer role="contentinfo">
-    <div class="container">
-        <div>
-          {% set lastPost = collections.posts | head(-1) %}
-            <p class="text-muted credits"> Copyright © {{ lastPost[0].year }} - Geshan Manandhar</p>
+<footer class="bg-darkpurple mt-24 sm:px-4">
+    <div class=" max-w-6xl mx-auto flex flex-col md:flex-row sm:flex-col justify-between items-center sm:items-center
+            md:items-center py-10 ">
+        <div class="pb-0 sm:pb-2">
+            {% set lastPost = collections.posts | head(-1) %}
+            <ul class="flex">
+                <li class=" text-white hover:text-gray-300 text-lg pr-4 cursor-pointer font-semibold ">
+                    <a href="#">Home</a>
+                </li>
+                <li class=" text-white hover:text-gray-300 text-lg pr-4 cursor-pointer font-semibold ">
+                    <a href="#">Posts</a>
+                </li>
+                <li class=" text-white hover:text-gray-300 text-lg cursor-pointer font-semibold ">
+                    <a href="#">Newsletter</a>
```

Comment:
possibly it is better to keep it in sync with header nav, wdyt?

Reply:
In sync as in Dynamic? We can do that

Reply:
yes.